File: src/main/java/me/lokka30/levelledmobs/rules/MobCustomNameStatus.java
/*
 * Copyright (c) 2020-2021  lokka30. Use of this source code is governed by the GNU AGPL v3.0 license that can be found in the LICENSE.md file.
 */

package me.lokka30.levelledmobs.rules;

/**
 * Holds values parsed from rules.yml
 *
 * @author stumper66
 */
public enum MobCustomNameStatus {
    NOT_SPECIFIED,  // default
    NAMETAGGED,     // Mob must be nametagged for the rule to work
    NOT_NAMETAGGED, // Mob must be not nametagged for the rule to work
    EITHER          // Doesn't matter what the nametag status of the mob is
}

File: levelledmobs-plugin/src/main/kotlin/io/github/arcaneplugins/levelledmobs/enums/MobCustomNameStatus.kt
package io.github.arcaneplugins.levelledmobs.enums

/**
 * Holds values parsed from rules.yml
 *
 * @author stumper66
 * @since 3.0.0
 */
enum class MobCustomNameStatus {
    NOT_SPECIFIED,  // default
    NAMETAGGED,  // Mob must be nametagged for the rule to work
    NOT_NAMETAGGED,  // Mob must be not nametagged for the rule to work
    EITHER // Doesn't matter what the nametag status of the mob is
}
